meeting-notes/2024-01-10.md

```diff
@@ -0,0 +1,31 @@
+# SRT meeting 2024-01-11
+
+Previous meeting notes: https://github.com/haskell/security-advisories/blob/main/meeting-notes/2023-12-13.md
+
+
+## 2023 H2 report
+
+- Draft sent to list; thanks for reviews.  FT will publish today.
+
+## ZuriHac plans
+
+- We agree it's a good idea to have a project, e.g. `cabal audit`, Hackage server.
+- Timeline: Jan for concept, March for concrete budget.
+- Jose has contact points with cabal-install and HLS.  hackage-server seems somewhat unloved.
+- Maybe we prioritise getting hackage-server attention?
+  - Many security improvment should/could be done (e.g. 2FA)
+- Can continue the discussion on list or GH issue (public).
+
+## Oustanding PRs
+
+- CWE library support.
+
+## Downstream toolling
+
+- Tristan already started something regarding tracking function calls 
+https://github.com/TristanCacqueray/cabal-audit
```

Comment:
I meant to mention that the issue with this approach is that, presently, inlined functions are hidden from the graph, so this project is blocked until a workaround is found or implemented. For example, perhaps GHC could add provenance information when code is inlined?

Reply:
I would start by investigating profiling SCC options, or DWARF info, to see if there is a reliable way to propagate the needed information through the build.